### PR TITLE
Move "Build, Deploy, and Sell Your Own Dynamic NFT" to the VRF section.

### PR DESCRIPTION
**Pull request by dwightjl**
Issue https://github.com/smartcontractkit/documentation/issues/310
Move "Build, Deploy, and Sell Your Own Dynamic NFT" to the VRF section, which might be a more applicable section for this content.

### CODE REVIEW

File: _data/navigation.js [hunk 1]
```diff
@@ -162,8 +162,8 @@ module.exports = {
                 url: '/docs/advanced-tutorial/',
               },
               {
-                title: 'Developer Tutorials',
-                url: '/docs/developer-tutorials/',
+                title: 'Tutorial Reference',
```
Comment (dwightjl) [inline _data/navigation.js:165]: I'm not completely satisfied with "Tutorial Reference" as a title, but for now that's what I suggest.

Comment (StephenFluin) [reply _data/navigation.js:165]: What about "Other Tutorials"?

Comment (StephenFluin) [reply _data/navigation.js:165]: I'm also not certain it deserves top billing in the introduction section, vs the reference sections lower in the navigation.

Comment (dwightjl) [reply _data/navigation.js]: The challenge here is that we have only one Tutorials section and right now its in our introduction. Long-term we might distinguish between "Getting Started" and "Tutorials", which would resolve the question of where this content should live in the navigation.

"Other Tutorials" is fine as a title for now. It flows acceptably out of the beginner tutorials, I think.

**Review by dmitrydao — APPROVED** [review]
LGTM

**Pull request merged**